src/map.cpp

```diff
@@ -1132,11 +1161,22 @@ bool map::displace_vehicle( vehicle &veh, const tripoint &dp )
                          "passenger at %d,%d,%d", prt, part_pos.x, part_pos.y, part_pos.z,
                          psg->posx(), psg->posy(), psg->posz() );
             }
+            const vehicle_part &veh_part = veh.part( prt );
+
+            // ramps make everything super tricky
+            int psg_offset_z = -ramp_offset;
+            tripoint next_pos = tripoint_zero;
```

Comment:
```suggestion
            tripoint next_pos;
```

Reply:
I'd prefer to leave this explicit, if that's okay.

Reply:
It is causing clang tidy error:

```
2020-06-29T12:25:42.3712358Z /home/runner/work/Cataclysm-DDA/Cataclysm-DDA/src/map.cpp:1165:33: error: Unnecessary initialization of 'next_pos'. 'tripoint' is zero-initialized by default. [cata-point-initialization,-warnings-as-errors]
2020-06-29T12:25:42.3712833Z             tripoint next_pos = tripoint_zero;
2020-06-29T12:25:42.3713166Z             ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~
2020-06-29T12:25:42.3713489Z             tripoint next_pos
```

Maybe we need to suppress it then.

Reply:
I think
```suggestion
            // NOLINTNEXTLINE(cata-point-initialization)
            tripoint next_pos = tripoint_zero;
```
would do it.

Reply:
I'll go fix it then.  Stupid clang.